StyleCop.Analyzers/StyleCop.Analyzers/LayoutRules/SA1513ClosingCurlyBracketMustBeFollowedByBlankLine.cs

```diff
@@ -249,6 +249,13 @@ private void AnalyzeCloseBrace(SyntaxToken token)
                         return;
                     }
 
+                    if (nextToken.IsKind(SyntaxKind.ColonToken)
+                        && IsPartOf<ConditionalExpressionSyntax>(token))
```

Comment:
I'm trying to think of a case where `}` is followed by `:` but is not part of a conditional expression.

Places a colon can appear:

* Conditional expressions
* Labels and switch cases (can't be preceded by `}`)
* Named arguments (can't be preceded by `}`)
* The target of an attribute (can't be preceded by `}`)
* Base type list (can't be preceded by `}`)
* Initializer expression (can't be preceded by `}`)

Are there any others?